op-e2e/l2blobs/l2blob_test.go

```diff
@@ -1,4 +1,4 @@
-package blobs
+package l2blobs
```

Comment:
Let's move it to `l2blob` so that the naming convention is consistent with `l2blob_test.go`.